easybuild/easyconfigs/s/snptest/snptest-2.5.6.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'Tarball'
+
+name = 'snptest'
```

Comment:
The documentation uses `SNPTEST` consistently for the name. Should we also use that?

Reply:
Yes, you're right. I was going on an old internal module that used lowercase.